apis/kueue/v1beta1/workload_types.go

```diff
@@ -347,11 +347,11 @@ type WorkloadStatus struct {
 	// +kubebuilder:validation:MaxItems=8
 	ResourceRequests []PodSetRequest `json:"resourceRequests,omitempty"`
 
-	// accumulatedPastExexcutionTimeSeconds holds the total time, in seconds, the workload spent
+	// accumulatedPastExecutionTimeSeconds holds the total time, in seconds, the workload spent
 	// in Admitted state, in the previous `Admit` - `Evict` cycles.
 	//
 	// +optional
-	AccumulatedPastExexcutionTimeSeconds *int32 `json:"accumulatedPastExexcutionTimeSeconds,omitempty"`
+	AccumulatedPastExecutionTimeSeconds *int32 `json:"accumulatedPastExecutionTimeSeconds,omitempty"`
```

Comment:
Oh 🤦 

Unfortunately, we can not fix this typo since this change brings cluster disruption caused by incompatible CRD.

Reply:
cc: @mimowo

Reply:
I recorded this fixes in v1beta2 wishlist: https://github.com/kubernetes-sigs/kueue/issues/768#issuecomment-2665905543

Thank you for finding this typo!
/close

Reply:
Or, we might be able to fix only the Go-typed name.
@wdyt?

Reply:
Or, we might be able to fix only the Go-typed name.
@mimowo wdyt?

Reply:
/reopen

Reply:
Nah, I don't think fixing just the go-typed name makes it better. 
However, I'm wondering what could go wrong if we merged it, because Workload API is just an internal API, but I'm ok to leave it for v1beta2. It will be an additional motivation to move forward with it.
cc @mwysokin @mwielgus to consider including v1beta2 in priorities for 0.12

Reply:
> Nah, I don't think fixing just the go-typed name makes it better.
> However, I'm wondering what could go wrong if we merged it, because Workload API is just an internal API, but I'm ok to leave it for v1beta2. It will be an additional motivation to move forward with it.
> cc @mwysokin @mwielgus to consider including v1beta2 in priorities for 0.12

That makes sense. So for now, let us close this PR.
@alexandear Thank you for reporting this!
/close